static/app/gettingStartedDocs/javascript/solid.tsx

```diff
@@ -88,11 +88,15 @@ ${getFeedbackConfigOptions(params.feedbackOptions)}}),`
 }
 });
 
-const app = document.getElementById("app");
+const root = document.getElementById("root");
 
-if (!app) throw new Error("No #app element found in the DOM.");
+if (import.meta.env.DEV && !(root instanceof HTMLElement)) {
```

Comment:
why do we need to check for `import.meta.env.DEV`? do solid projects always have `env`?

Reply:
This code is included in the solid boilerplate. We are just updating the snippet to match it.